pkg/eval/builtin_special.go

```diff
@@ -41,8 +41,17 @@ var IsBuiltinSpecial = map[string]bool{}
 // NoSuchModule encodes an error where a module spec cannot be resolved.
 type NoSuchModule struct{ spec string }
 
+// ModuleLoadError encapsulates an error message a failed module load operation.
+type ModuleLoadError struct {
```

Comment:
This should probably be `PluginLoadError` because it's not used for Elvish modules.